test/runtests.jl

```diff
@@ -1,20 +1,22 @@
 using LinearAlgebra
 using SparseArrays
 using Test
+using Random
 
 using AMD
 import AMD: AMD_STATUS, AMD_OK, COLAMD_STATUS, COLAMD_OK
 
-_Clong = Base.Sys.WORD_SIZE == 32 ? Clong : Clonglong
+Random.seed!(666)
+
 for n in [10, 20, 30]
   for density in [0.25, 0.75, 1.0]
-    for T in [Cint, _Clong]
+    for T in [Cint, Int64]
       A = convert(SparseMatrixCSC{Float64, T}, sprand(n, n, density))
       @test amd_valid(A)
 
       meta = Amd()
       p = amd(A, meta)
-      @test meta.info[AMD_STATUS] == AMD_OK
+      @test meta.info[AMD_STATUS+1] == AMD_OK
```

Comment:
Why the `+1` here? It would be more natural to add 1 to the constants in the `#define`s.

Reply:
Because I would like to not modify the files generated by Clang.jl.
We will need to modify by hand some constants if we regenerate the wrappers for future releases of SuiteSparse.

It only concerns 4 files so that's fine but we must add a note in the README of the gen folder to not forget it.

Reply:
I agree, we should hide the `+1`. Do you have an example of a `get_info()` function that you have in mind?
I suppose that you want that the function takes a symbol as input.

Reply:
I added +1 to all constants that give a position in a vector. We should be good now.
I forgot to finalize this PR last month.